### ISSUE TITLE
Remove duplication in browserslist

### ISSUE THREAD

**Issue by mewdriller** [b2io/base2.io#207]
Currently there is a duplicated list of target browsers in both `.stylelintrc` and `.babelrc`. This can be avoided using the [browserslist](https://github.com/ai/browserslist) field in `package.json` or a separate `browserslist` file.

**Comment by csteinert** [comment]
External references won't be supported in `babel-preset-env` until v7.0.0 (currently v7.0.0-beta.39) so we can't do this just yet. I closed the PR but left the branch. Once it's out of beta, we're good to try this again.

**Comment by csteinert** [comment]
The repo for `babel-preset-env` has moved, but it is still in beta. Is now here: https://github.com/babel/babel/tree/master/packages/babel-preset-env

**Comment by tborres** [comment]
Previous task description and back-and-forth:

Currently there is a duplicated list of target browsers in both `.stylelintrc` and `.babelrc`. This can be avoided using the [browserslist](https://github.com/ai/browserslist) field in `package.json` or a separate `browserslist` file.

~External references won't be supported in `babel-preset-env` until v7.0.0 (currently v7.0.0-beta.39) so we can't do this just yet. I closed the PR but left the branch. Once it's out of beta, we're good to try this again.~

☝️ This package has now been updated.
The repo for `babel-preset-env` has moved, ~but it is still in beta.~ Is now here: https://github.com/babel/babel/tree/master/packages/babel-preset-env
